functions/src/api/v1/reminders/createReminder.ts

```diff
@@ -58,15 +61,41 @@ import { assertTimestampInFuture } from './assertTimestampInFuture'
  *       412:
  *         description: authorization header present but not formatted correctly
  */
-export const createReminder = wrap(async (req: Request, res: Response) => {
+export const createReminderRoute = wrap(async (req: Request, res: Response) => {
     const userId = res.locals.uid
     const roomId = req.params.roomId
+    const { hostName, destinations, sendAt } = req.body
+
+    const reminderId = await createReminder({
+        roomId,
+        userId,
+        hostName,
+        sendAtParameter: sendAt,
+        destinationsParameter: destinations,
+    })
+
+    res.send({
+        reminderId,
+    })
+})
+
+export const createReminder = async ({
+    roomId,
+    userId,
+    sendAtParameter,
+    hostName,
+    destinationsParameter,
+}: {
+    roomId: RoomId
+    userId: UID
+    sendAtParameter: string
+    hostName: string
+    destinationsParameter: string
+}): Promise<ReminderId> => {
     await assertRightToEditRoom(roomId, userId)
 
-    const body = req.body
-    const sendAt = parseInt(body.sendAt) * 1000
-    const hostName = req.body.hostName
-    const destinations = JSON.parse(body.destinations || '[]')
+    const sendAt = parseInt(sendAtParameter) * 1000
+    const destinations = JSON.parse(destinationsParameter || '[]')
```

Comment:
`JSON.parse` should be wrapped in order to provide the API consumer with a more specific error (like wrong arguments format)